source/examples/header/header-sample095.txt

```diff
@@ -0,0 +1 @@
+<plateNum>A & P. No. 6412</plateNum>
```

Comment:
This throws the mentioned ERROR concerning the wellformedness:

```suggestion
<plateNum>A &amp; P. No. 6412</plateNum>
```

will fix it!

all credits for this go to @musicEnfanthen  ;-)